docs/usage.md

```diff
@@ -2,45 +2,60 @@
 
 ## Table of contents
 
-* [Introduction](#general-nextflow-info)
+* [Table of contents](#table-of-contents)
+* [Introduction](#introduction)
 * [Running the pipeline](#running-the-pipeline)
-* [Updating the pipeline](#updating-the-pipeline)
-* [Reproducibility](#reproducibility)
+  * [Updating the pipeline](#updating-the-pipeline)
+  * [Reproducibility](#reproducibility)
 * [Main arguments](#main-arguments)
-    * [`-profile`](#-profile-single-dash)
-        * [`docker`](#docker)
-        * [`awsbatch`](#awsbatch)
-        * [`standard`](#standard)
-        * [`binac`](#binac)
-        * [`cfc`](#cfc)
-        * [`none`](#none)
-    * [`--reads`](#--reads)
-    * [`--singleEnd`](#--singleend)
-* [Reference Genomes](#reference-genomes)
-    * [`--genome`](#--genome)
-    * [`--fasta`](#--fasta)
-* [Job Resources](#job-resources)
-* [Automatic resubmission](#automatic-resubmission)
-* [Custom resource requests](#custom-resource-requests)
-* [AWS batch specific parameters](#aws-batch-specific-parameters)
-    * [`-awsbatch`](#-awsbatch)
-    * [`--awsqueue`](#--awsqueue)
-    * [`--awsregion`](#--awsregion)
+  * [`-profile`](#-profile)
+  * [`--input`](#--input)
+* [Reference genomes](#reference-genomes)
+  * [`--genome` (using iGenomes)](#--genome-using-igenomes)
+  * [`--fasta`](#--fasta)
+  * [`--bed`](#--bed)
+  * [`--mapping`](#--mapping)
+  * [`--vcf`](#--vcf)
+  * [`--igenomes_ignore`](#--igenomes_ignore)
+* [Job resources](#job-resources)
+  * [Automatic resubmission](#automatic-resubmission)
+  * [Custom resource requests](#custom-resource-requests)
+* [AWS Batch specific parameters](#aws-batch-specific-parameters)
+  * [`--awsqueue`](#--awsqueue)
+  * [`--awsregion`](#--awsregion)
+  * [`--awscli`](#--awscli)
+* [Processing parameters](#processing-parameters)
+  * [`--trim5`](#--trim5)
+  * [`--polyA`](#--polyA)
+  * [`--multimappers`](#--multimappers)
+  * [`--quantseq`](#--quantseq)
+  * [`--endtoend`](#--endtoend)
+  * [`--min_coverage`](#--min_coverage)
+  * [`--var_fraction`](#--var_fraction)
+  * [`--conversions`](#--conversions)
+  * [`--baseQuality`](#--baseQuality)
+  * [`--readLength`](#--readLength)
+  * [`--pvalue`](#--pvalue)
+  * [`--skipTrimming`](#--skipTrimming)
```

Comment:
We try to stay consistent with `snake_case` for parameters where possible..

```suggestion
  * [`--skip_trimming`](#--skip_trimming)
```

Reply:
Yes I overlooked those.